content/blog/2016/2016-03-11-new-jenkins-20-preview.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "New Jenkins 2.0 Preview Build"
+tags:
+- jenkins2
+author: daniel-beck
+---
+
+We just published the new Jenkins 2.0 preview build alpha 3.
+
+## What's new?
+
+* **Jenkins is now secure out of the box**:
+  Administrators previous had to set up authentication and authorization while Jenkins was accessible to anyone on the same network.
+  Now, Jenkins is protected out of the box, so that it is always safe from unauthorized access.
+* **Plugin selection for setup**:
+  We refined the plugin selection on the setup dialog.
+  You've always wondered why Jenkins does not install the Git Plugin by default?
+  Now it does.
+  We're also including more plugins improving [Pipeline](/solutions/pipeline/):
+  The [Pipeline Stage View](https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Stage+View+Plugin) plugin lets you quickly see what's going on in your CD pipeline, and the [GitHub Organization Folder Plugin](https://github.com/jenkinsci/github-organization-folder-plugin#github-organization-folder) allows you to set up a job container that automatically sets itself up with jobs for all Jenkins pipelines defined in any repo of your GitHub organization.
```

Comment:
I'd split this one sentence in two.

"allows you to set up a job container that automatically sets itself up with jobs for all Jenkins pipelines defined in any repo of your GitHub organization" feels a bit unwieldy.

Maybe something like: "can scan your GitHub organization for repositories that have defined a Jenkins Pipeline, and will automatically create new Jenkins jobs for them"